pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+dynamic = ["version"]
+name = "tomato-alicat"
+authors = [
+  {name="Alexandre Gbocho", email="gbocho@tu-berlin.de"},
+  {name="Peter Kraus", email="peter.kraus@ceramics.tu-berlin.de"},
+]
+
+description = "This package is a part of tomato. It has been made for the automation of alicat devices."
+readme = "README.md"
+requires-python = ">=3.9"
+dependencies = [
+    "alicat,
+    "tomato >= 1.0rc1",
```

Comment:
Syntax error here. A minimum version for `alicat` would be good.

You can depend on `tomato >= 1.0` since that was released already.